### ISSUE TITLE
Validation of the whl fails 

### ISSUE THREAD

**Issue by rahedges** [NVIDIA/nvImageCodec#2]
### Version

cu12-0.2.0.7-py3-none-manylinux2014_x86_64

### Describe the bug.

When installing the python package, this warning is generated:

```
Warning: Validation of the RECORD file of nvidia_nvimgcodec_cu12-0.2.0.7-py3-none-manylinux2014_x86_64.whl failed. Please report to the maintainers of that package so they can fix their build process. Details:
In /home/fake-user/.cache/pypoetry/artifacts/47/fd/7c/afc85ea572653ace895bbd45b706d8ba5036a63e863dd0c0d773161220/nvidia_nvimgcodec_cu12-0.2.0.7-py3-none-manylinux2014_x86_64.whl, nvimgcodec_rec.lock is not mentioned in RECORD

```

### Minimum reproducible example

```shell
Create this pyproject.toml file:

[tool.poetry]
name = "nvimgcodec-test"
version = "0.1.0"
description = ""
authors = ["anon <anon@anon.com>"]
readme = "README.md"

[tool.poetry.dependencies]
python = ">=3.9,<=3.12"
nvidia-nvimgcodec-cu12 = "^0.2.0.7"


[build-system]
requires = ["poetry-core"]
build-backend = "poetry.core.masonry.api"
```

run this command:
`poetry install --no-root`
```


### Environment details

_No response_

### Relevant log output

```shell
Warning: Validation of the RECORD file of nvidia_nvimgcodec_cu12-0.2.0.7-py3-none-manylinux2014_x86_64.whl failed. Please report to the maintainers of that package so they can fix their build process. Details:
In /home/fake-user/.cache/pypoetry/artifacts/47/fd/7c/afc85ea572653ace895bbd45b706d8ba5036a63e863dd0c0d773161220/nvidia_nvimgcodec_cu12-0.2.0.7-py3-none-manylinux2014_x86_64.whl, nvimgcodec_rec.lock is not mentioned in RECORD
```

### Check for duplicates

- [X] I have searched the [open bugs/issues](https://github.com/NVIDIA/nvImageCodec/issues) and have found no duplicates for this bug report

**Comment by jantonguirao** [comment]
Thank you @rahedges for reporting this. We'll add a fix for it on the next release.

**Comment by jantonguirao** [comment]
This issue should be fixed with the [new release v0.3.0](https://github.com/NVIDIA/nvImageCodec/releases/tag/v0.3.0). Please reopen if the issue persists.
